Src/DependencyCollector/Net46.Tests/DependencyTrackingTelemetryModuleTestNet46.cs

```diff
@@ -270,7 +342,7 @@ public void OnBeginOnEndAreNotCalledForAppInsightsUrl()
         private void ValidateTelemetryForDiagnosticSource(DependencyTelemetry item, Uri url, WebRequest request, bool success, string resultCode)
         {
             Assert.AreEqual(url, item.Data);
-            Assert.AreEqual(url.Host, item.Target);
+            Assert.AreEqual($"{url.Host}:{url.Port}", item.Target);
```

Comment:
I'm afraid to use localhost:80 in tests, I suspect the port could be used in multiple cases on build and developers machines.
I'll use it in dns issue test, so we'll verify it.
It's checked with FrameworkHttpProcessingTest for event source